src/main/java/com/couchbase/lite/support/JsonDocument.java

```diff
@@ -36,7 +36,13 @@ public Object jsonObject() {
                 tmp = new LazyJsonArray<Object>(json);
             } else {
                 try {
-                    tmp = Manager.getObjectMapper().readValue(json, Object.class);
+                    // this is for Jackson 2.5.0
```

Comment:
ok that makes sense.  can you add that to the comments?

so does that mean we are giving invalid json to Jackson?